internal/core/application/wallet_service.go

```diff
@@ -381,6 +381,18 @@ func sendToMany(opts sendToManyOpts) (string, error) {
 		return "", err
 	}
 
+	inputBlindingData := make(map[int]wallet.BlindingData)
+	index := 0
+	for _, v := range updateResult.SelectedUnspents {
+		inputBlindingData[index] = wallet.BlindingData{
+			Asset:         v.Asset(),
+			Amount:        v.Value(),
+			AssetBlinder:  v.AssetBlinder(),
+			AmountBlinder: v.ValueBlinder(),
+		}
+		index++
+	}
```

Comment:
```suggestion
	inputBlindingData := make(map[int]wallet.BlindingData, 0)
	for _, v := range updateResult.SelectedUnspents {
		inputBlindingData = append(inputBlindingData, wallet.BlindingData{
			Asset:         v.Asset(),
			Amount:        v.Value(),
			AssetBlinder:  v.AssetBlinder(),
			AmountBlinder: v.ValueBlinder(),
		})
	}
```

Reply:
@altafan how can we use append with map?

Reply:
sorry I misread it 🙏. Resolving.